src/backends/onnxruntime.c

```diff
@@ -225,9 +237,15 @@ RAI_Model *RAI_ModelCreateORT(RAI_Backend backend, RAI_Device device,
   }
 
   // TODO: probably these options could be configured at the AI.CONFIG level
-  OrtSessionOptions* session_options = OrtCreateSessionOptions();
-  OrtSetSessionThreadPoolSize(session_options, 1);
-  OrtSetSessionGraphOptimizationLevel(session_options, 1);
+  OrtSessionOptions* session_options;
+  status = OrtCreateSessionOptions(&session_options);
+  if (status != NULL) goto error;
```

Comment:
Do we need to free env if we fail here?

Reply:
Yep, and session_options a bit down. Fixed.

Reply:
Sorry, taking it back. `env` is global, it is only allocated the first time and it shouldn't be freed.